maat-court-data-api/src/main/java/gov/uk/courtdata/eform/controller/EformStagingController.java

```diff
@@ -87,4 +87,18 @@ public ResponseEntity<Void> createEformApplication(@PathVariable Integer usn,
 
         return ResponseEntity.ok().build();
     }
+
+    @PostMapping(value ="/validate/{usn}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Create an EFORMS_STAGING record for Crime Apply")
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
+    @ApiResponse(responseCode = "400", description = "Bad Request.", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ErrorDTO.class)))
+    @ApiResponse(responseCode = "500", description = "Server Error.", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ErrorDTO.class)))
+    public ResponseEntity<EformStagingResponse> retriveOrInsertUsnRecord(@PathVariable Integer usn) {
```

Comment:
Would maybe consider renaming this method from `retriveOrInsertUsnRecord` to `retriveOrInsertDummyUsnRecord` I think we should draw attention to the fact that if we do create a USN record, it will only be a dummy/placeholder record and won't be fully populated? Am open to ideas, what do you @skasthuri36326 think?

Reply:
It is a placeholder record. We can call it as retriveOrInsertDummyUsnRecord to be explicit.

Reply:
sure, sounds good